master/usage/configuration/bgp.md

```diff
@@ -144,27 +180,34 @@ spec:
 EOF
 ```
 
-To remove a global BGP peer that was configured with IP address 192.20.30.40
-(the AS number is not required when deleting) run the following command on any
-node:
+To view the current list of BGP peers run the following command.
 
-	$ calicoctl delete bgpPeer 192.20.30.40 --scope=global
+```
+calicoctl get bgpPeer 
+```
 
-To view the current list of global peers run the following command on any node:
+It should return something like the following.
 
 ```
-$ calicoctl get bgpPeer --scope=global
-SCOPE    PEERIP         NODE   ASN
-global   192.20.30.40          64567
+NAME                  PEERIP         NODE      ASN
+bgppeer-global-3040   192.20.30.40   (global)  64567
 ```
 
-### Configuring a node specific BGP peer
+To remove the global BGP peer that you just created run the following command on any
+node:
+
+```
+calicoctl delete bgppeer bgppeer-node-aabbff
```

Comment:
this should be `bgppeer-global-3040` (the one you just created)

Reply:
Ooops, thanks @gunjan5